src/widgets/Header/Header.module.scss

```diff
@@ -0,0 +1,134 @@
+@import '../../styles/variables.scss';
+@mixin onTablet {
```

Comment:
Please move mixin to shared/styles/mixin.scss and then @import that file here and use with @include mixinName, because probably we will use them a lot